exotica_python/src/pyexotica.cpp

```diff
@@ -960,6 +960,14 @@ PYBIND11_MODULE(_pyexotica, module)
     scene.def("get_model_joint_names", &Scene::GetModelJointNames);
     scene.def("get_model_state", &Scene::GetModelState);
     scene.def("get_model_state_map", &Scene::GetModelStateMap);
+    scene.def("get_tree_names", [](Scene& scene) {
+        std::vector<std::string> frame_names;
+        for (const auto& m : scene.GetTreeMap())
+        {
+            frame_names.push_back(m.first);
```

Comment:
Let's try to use `emplace_back` in new code.

Reply:
`emplace_back` is constructing an object in place. I think in this case, `emplace_back` would just call the string copy constructor, which effectively is the same as `push_back` the object.